src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.page {
+  margin: 0;
+  padding-inline: 8px;
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin: 20px 0 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+/* .search-bar--small {
+  margin-top: 20px;
+} */
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:active {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-bar__form {
+  display: flex;
+  align-items: center;
+}
+
+.search-bar__form--big {
+  height: 70px;
+}
+
+.search-bar__form--small {
+  height: 42px;
+}
+
+.icon {
+  background-position: center;
+  background-size: cover;
+  color: #687c94;
+}
+
+.icon--big {
+  height: 19px;
+  width: 19px;
+  margin: 0 16px 0 26px;
+}
+
+.icon--small {
+  height: 11px;
+  width: 11px;
+  margin: 0 9px 0 13px;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  border: 0 solid;
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.input--small {
+  align-items: center;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.input:focus {
+  font-family: Avenir, sans-serif;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 900;
+  line-height: normal;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
you use this background twice, so lets create a variable for it. does it makes sense?